__tests__/Ship.test.js

```diff
@@ -0,0 +1,64 @@
+/* globals describe it expect */
+const Ship = require('../src/Ship.js')
+const Port = require('../src/Port.js')
```

Comment:
Since you are not creating any instances of `Port` in this test this import can be deleted